index.html

```diff
@@ -462,6 +462,13 @@
                 '<p>$2</p>' +
                 '<p><em>$3</em></p>' +
                 '<p class="helptext__stats">$4</p>');
+            if (parseInt(id) === 276) {
+                // Handle Wonders separately as the upgrades text is missing for them.
```

Comment:
Do we want to be brave here and not check for a specific ID, but for the absence of `<i>`?

Reply:
Makes sense. That way if the info is fixed for Wonders, it would not require any changes.